ledger/metrics/src/main/scala/com/daml/telemetry/Telemetry.scala

```diff
@@ -30,6 +31,14 @@ abstract class Telemetry(protected val tracer: Tracer) {
     */
   def contextFromMetadata(metadata: Option[jMap[String, String]]): TelemetryContext
 
+  /** Allows to transform an Open Telemetry context into a telemetry context.
+    * Makes integration easier for consumers that are using the Open Telemetry API directly.
+    * Consider using other factory methods.
+    *
+    * @param context raw Open Telemetry context
+    */
+  def contextFromOpenTelemetryContext(context: Context): TelemetryContext
```

Comment:
This is a bit shorter and it works too IMO, as the parameter type/name already contains the info:

```suggestion
  def contextFromOpenTelemetry(context: Context): TelemetryContext
```

Reply:
@fabiotudone-da I wanted to keep it consistent with the previous ones like `contextFromGrpcThreadLocalContext`. Does it make sense or shall I rename it?

Reply:
Yes sure, it's just a minor thing.

Reply:
Another option is to name this explicitly `adaptOpenTelemetryContext` (to avoid having to repeat 'context') and leave the other one constructing a context from gRPC thread-local context as-is.